addons/l10n_ae/data/account.account.template.csv

```diff
@@ -1,183 +1,339 @@
-"id","name","code","user_type_id/id","chart_template_id/id","reconcile"
-"uae_account_3665","Main Safe","101111","account.data_account_type_current_assets","l10n_ae.uae_chart_template_standard","False"
```

Comment:
Please use the same format for the csv as we had before, so we see what has actually been changed in the CoA structure.

Reply:
We are setting a nocreate = True for all the previous accounts and creating new ones from scratch with different ids, codes and sometimes names and types.

Reply:
I don't think we can adapt the existing one especially that I am changing all the record ids as per the suggestion of WAN. The reason why it is like that is to not break the current stage version also as per WAN. So do you think I should just keep the new accounts and make a PR to v15 instead?

Reply:
Remove old accounts